pages/src/TodoList.js

```diff
@@ -114,6 +124,8 @@ export default class TodoList extends Component {
                   ? <del>{label}</del>
                   : label
                 }
+                {' '}
+                <button type='button' onClick={(evt) => this._onDeleteElmntTodo(evt, id)}>{' '}X{' '}</button>
```

Comment:
Inspire yourself from the above `checkbox` and avoid creating an intermediary function for `onClick`.